src/core/SampleBuffer.cpp

```diff
@@ -1403,8 +1380,7 @@ QString SampleBuffer::tryToMakeRelative( const QString & file )
 {
 	if( QFileInfo( file ).isRelative() == false )
 	{
-		// Normalize the path
-		QString f = QFileInfo( file ).canonicalFilePath().replace( QDir::separator(), '/' );
+		QString f = QString( file ).replace( QDir::separator(), '/' );
```

Comment:
You should revert this change. FYI, this part is changed again in `stable-1.2` and will by synced back into `master`.

Reply:
Thanks! I wasn't sure what I did wrong.  Done.